addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -1039,10 +1039,10 @@ registerWebsitePreviewTour("website_form_editable_content", {
     edition: true,
 }, () => [
     {
-        trigger: ".o_website_preview.editor_enable.editor_has_snippets",
+        trigger: ".o_website_preview :iframe .odoo-editor-editable",
```

Comment:
To check whether snippets are there we should check .o_snippet class which is available in snippet-menu